sql/core/src/test/resources/tpcds-plan-stability/approved-plans-modified/q10.sf100/simplified.txt

```diff
@@ -1,81 +1,100 @@
 TakeOrderedAndProject [cd_gender,cd_marital_status,cd_education_status,cd_purchase_estimate,cd_credit_rating,cd_dep_count,cd_dep_employed_count,cd_dep_college_count,cnt1,cnt2,cnt3,cnt4,cnt5,cnt6]
-  WholeStageCodegen (14)
+  WholeStageCodegen (20)
     HashAggregate [cd_gender,cd_marital_status,cd_education_status,cd_purchase_estimate,cd_credit_rating,cd_dep_count,cd_dep_employed_count,cd_dep_college_count,count] [count(1),cnt1,cnt2,cnt3,cnt4,cnt5,cnt6,count]
       InputAdapter
         Exchange [cd_gender,cd_marital_status,cd_education_status,cd_purchase_estimate,cd_credit_rating,cd_dep_count,cd_dep_employed_count,cd_dep_college_count] #1
-          WholeStageCodegen (13)
+          WholeStageCodegen (19)
             HashAggregate [cd_gender,cd_marital_status,cd_education_status,cd_purchase_estimate,cd_credit_rating,cd_dep_count,cd_dep_employed_count,cd_dep_college_count] [count,count]
               Project [cd_gender,cd_marital_status,cd_education_status,cd_purchase_estimate,cd_credit_rating,cd_dep_count,cd_dep_employed_count,cd_dep_college_count]
-                BroadcastHashJoin [c_current_cdemo_sk,cd_demo_sk]
+                SortMergeJoin [c_current_cdemo_sk,cd_demo_sk]
```

Comment:
unexpected change @cloud-fan

Reply:
Looks interesting. Why did the plan change here? The plan cannot correctly get stats from a table with char/varchar?

Reply:
I guess so. The padding expression stops stats propagation in CBO and we need to fix that.

Reply:
I gonna take a look at this

Reply:
Oh..